misc/formatter/modules/formatter-core/src/test/resources/actions/type-cast/assert/type_cast_action_1.bal

```diff
@@ -1,15 +1,9 @@
-type Person record {
-    string name;
-};
+type Person record { string name; };
```

Comment:
I think we should expand module-level record type defs always.

I have fixed this with https://github.com/ballerina-platform/ballerina-lang/commit/51673806e91612de5f6aca3da9f0d59dadea07a2